client/post-editor/editor-drawer/taxonomies.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import map from 'lodash/map';
+
+/**
+ * Internal dependencies
+ */
+import QueryTaxonomies from 'components/data/query-taxonomies';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { getEditedPostValue } from 'state/posts/selectors';
+import { getPostTypeTaxonomies } from 'state/post-types/taxonomies/selectors';
+import Accordion from 'components/accordion';
+import Gridicon from 'components/gridicon';
+
+function EditorDrawerTaxonomies( { siteId, postType, taxonomies } ) {
+	return (
+		<div className="editor-drawer__taxonomies">
+			{ siteId && postType && (
+				<QueryTaxonomies { ...{ siteId, postType } } />
+			) }
+			{ map( taxonomies, ( taxonomy ) => {
+				if ( 'post_format' === taxonomy.name ) {
+					// Post format has its own dedicated accordion
+					return;
+				}
+
+				return (
+					<Accordion
+						key={ taxonomy.name }
+						title={ taxonomy.label }
+						icon={ <Gridicon icon="tag" /> }>
```

Comment:
I actually think keeping the affordances that are in place might be the most logical way to go: folder icon for hierarchical taxonomies (category-based), tag icon for linear taxonomies (tag-based). The repetition only makes their purpose clearer, but in this specific case, "Project Types" should be the folder icon, and "Project Tags" should remain the tag icon. 

Any thoughts @mtias?